### DISCUSSION TITLE
Readonly doesn't work

### DISCUSSION THREAD

**Discussion by halivert** [vuejs/petite-vue#99] [Q&A] [answered]
How to use a variable to set `readonly` attribute in a textarea?

Example:
https://codepen.io/halivert/pen/gOxWxMW

**Comment by niutech** [comment, marked as answer]
It works, but you assign `true` or `false` value to `readonly`, whereas it should be `true` or `undefined` to remove the attribute. So type: `:readonly="readonly || undefined"`

**Reply by halivert** [reply]
Thank you so much 🙏🏽
